src/Mealz/MealBundle/Tests/Controller/MealControllerTest.php

```diff
@@ -339,13 +337,7 @@ public function testNewMealFlag(): void
         }
     }
 
-    /**
-     * Gets all the participants for a meal.
-     *
-     * @param Meal $meal Meal instance
-     * @return array
-     */
-    private function getMealParticipants($meal)
+    private function getMealParticipants(Meal $meal): array
```

Comment:
We can add "@return Participant[]" to PHP-Doc. It conveys more information than just simple array.

Reply:
Ok.